src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs

```diff
@@ -147,31 +98,57 @@ private SocketError ProcessIOCPResult(bool success, int bytesTransferred)
                     FinishOperationSyncSuccess(bytesTransferred, SocketFlags.None);
                     return SocketError.Success;
                 }
-
-                // Socket handle is going to post a completion to the completion port (may have done so already).
```

Comment:
Can we keep the comment here, or some version of it, to clarify why we're falling through here?

Reply:
Done